MeshLib/Elements/LineRule2.h

```diff
@@ -39,6 +39,9 @@ class LineRule2 : public EdgeRule
     /// Constant: The FEM type of the element
     static const CellType cell_type = CellType::LINE2;
 
+    /// Constant: The number of edges
+    static const unsigned n_edges = 1;
```

Comment:
Not sure here, because 1d entities can have only one type of boundary, which is 0d. Can you explain please why it is needed or what is wrong without that edge information?